### DISCUSSION TITLE
Thin plate spline

### DISCUSSION THREAD

**Discussion by Vrushank264** [kornia/kornia#1841] [General]
I am currently facing some issues in Kornia's thin plate spline.

I get a wrongly warped image with the correct source and destination points. I ran the same script on other publically available Tps implementations and they seem to correctly warp the image but they are painfully slow.

It would be really helpful if someone can make a colab notebook on how to properly use the thin_plate_spline functionality of Kornia with some example images.

**Comment by edgarriba** [comment]
hi ! could you share your current script ? in order to have a reference, you could check here 
https://github.com/kornia/kornia/blob/master/kornia/augmentation/_2d/geometric/thin_plate_spline.py#L54

/cc @catalys1

**Comment by catalys1** [comment]
@Vrushank264 Does this [basic tutorial](https://colab.research.google.com/drive/1B09C0ZoOBdm4lB-6OwPfcz-tjIdlz5wd) help?

**Reply by edgarriba** [reply]
@catalys1 would be great to have this tutorial in
https://kornia-tutorials.readthedocs.io/en/latest/index.html

i'm missign something here, like showing the keypoints warped too (i believe we supported that)
